unlock-app/src/hooks/useAccount.ts

```diff
@@ -84,6 +84,42 @@ export const useAccount = (address: string, network: number) => {
     }
   }
 
+  const disconnectStripeToLock = async ({
+    lockAddress,
+    network,
+  }: {
+    lockAddress: string
+    network: number
+  }) => {
+    const storageService = new StorageService(config.services.storage.host)
+    const typedData = generateTypedData(
```

Comment:
Hum, is your new API endpoint using this? If so, sorry I missed it but you should move to using token based auth